src/Modal.js

```diff
@@ -0,0 +1,170 @@
+// @flow
+
+import React from 'react';
+import classNames from 'classnames';
+import './Modal.scss';
+
+type ModalProps = {
```

Comment:
Our Modal component is based on the W3C example code and we should add a note to acknowledge that.

* https://www.w3.org/TR/wai-aria-practices-1.1/examples/dialog-modal/js/dialog.js
* https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document

Reply:
Thank you for the suggestion, I did not think about that. Please let me know if the acknowledgement I wrote is not enough.

Reply:
The license includes a template to use: "This software or document includes material copied from or derived from [title and URI of the W3C document]. Copyright © [YEAR] W3C® (MIT, ERCIM, Keio, Beihang)." Please use that:

```
// This software or document includes material copied from or derived from
// WAI-ARIA Authoring Practices 1.1 Modal Dialog Example
// https://www.w3.org/TR/wai-aria-practices-1.1/examples/dialog-modal/dialog.html
// Copyright © 2019 W3C® (MIT, ERCIM, Keio, Beihang).
// Source code: https://www.w3.org/TR/wai-aria-practices-1.1/examples/dialog-modal/js/dialog.js
// License: https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document
```

Reply:
And please also add a note and link to the license to the table at the bottom of our README.